.packit.yaml

```diff
@@ -102,5 +102,5 @@ jobs:
     packit_instances: ["prod"]
     metadata:
       dist_git_branches:
-        - fedora-stable # rawhide updates are created automatically
+        - fedora-latest # branched version, rawhide updates are created automatically
```

Comment:
wouldn't it make sense to have both here?

with fedora-latest, we won't create updates for other stable releases in case there are multiple (most of the time there are 2)

Reply:
Ah, copy-paste error.. Sorry.